portfolio/src/main/webapp/script.js

```diff
@@ -140,9 +140,19 @@ const mapStyle = [
 /**
  * Calls the methods that load the home page.
  */
-function start() {
-  getCommentSection();
+async function start() {
```

Comment:
BTW I realized I might have missed something in your last PR.  I believe you are including this script.js file from all your HTML files right?  However only `index.html` loads the Google Charts API `https://www.gstatic.com/charts/loader.js`.  I think this might cause some problems, because pages like pictures.html will still be running the code in `script.js` which will call `google.charts.load` but fail.

I think you could fix this by either:
1) having a separate JavaScript file with this code, maybe called something like `chart.js`
or
2) creating a function to load the charts, and only calling that function when chart.html loads.  Something like:
```
function loadChartsApi() {
  google.charts.load('current', {'packages':['corechart']});
  google.charts.setOnLoadCallback(drawChart);
}
```

Reply:
Thanks for catching this. I do see an error now in the console when I switch to other pages, even though they do display the correct content. I took the second approach, calling 'loadChartsApi()` in the 'start(page)' method when `page==='home'`